src/shared/report_op.c

```diff
@@ -83,7 +83,7 @@ static int _os_report_str_int_compare(const char *str, int id)
             }
             pt_check = 1;
         } else {
-            return (-1);
+            return -1;
```

Comment:
The rest of `return` calls for this function is written with brackets, change them to be consistent.